app/components/Settings/AccessSettings.jsx

```diff
@@ -44,18 +44,29 @@ class ApiNode extends React.Component {
 
         let color;
         let latencyKey;
+        let friendlyPing;
 
-        if(ping < 400) {
+        if (ping < 400) {
             color = "low";
             latencyKey = "low_latency";
-        }
-        else if(ping >= 400 && ping < 800) {
+        } else if (ping >= 400 && ping < 800) {
             color = "medium";
             latencyKey = "medium_latency";
         } else {
             color = "high";
             latencyKey = "high_latency";
         }
+
+        if (ping >= 1000) {
+            let pong = ping;
```

Comment:
I'm having a hard time understanding why all this math is necessary and what it accomplishes. The ping number is already in ms, all you need to do to get seconds is simply divide it by 1000.

I also don't think it's a good idea to reinstate the actual ping numbers, we used to have that and I removed them because people were getting hung up on the values being different from regular ping numbers which will always be the case.

Reply:
Simple division was my first choice, but I wanted to round to 2 decimal places. I can redo and handle the type conversions with type conversion and toFixed if you'd like.

But, if you don't want them in there at all, then I can pull the code.

Obviously, these aren't ICMP pings, but I think it's useful to know.